package.json

```diff
@@ -19,7 +21,7 @@
   ],
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged && npm run lint && npm test"
+      "pre-commit": "pretty-quick --staged && npm run lint && npm test && npm run typescript"
```

Comment:
If you want a perf boost to the pre-commit hook, see [here](https://github.com/openmrs/openmrs-esm-api/blob/954a83e1e19b145f4bb75a1c610de483150ef530/package.json#L21)